netty/src/main/java/io/grpc/netty/NettyClientTransport.java

```diff
@@ -183,20 +184,6 @@ public void operationComplete(ChannelFuture future) throws Exception {
     }).channel();
     // Start the write queue as soon as the channel is constructed
     handler.startWriteQueue(channel);
-    // This write will have no effect, yet it will only complete once the negotiationHandler
-    // flushes any pending writes.
-    channel.write(NettyClientHandler.NOOP_MESSAGE).addListener(new ChannelFutureListener() {
-      @Override
-      public void operationComplete(ChannelFuture future) throws Exception {
-        if (future.isSuccess()) {
```

Comment:
I'd feel better if we left this for now and just did nothing if `future.isSuccess()` is true (a.k.a., just don't call `transportReady()`). I do think we _could_ remove it, but then the transport won't have a good status code of what went wrong (which doesn't matter _today_, but...)